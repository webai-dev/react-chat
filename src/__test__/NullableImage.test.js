import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Nullable from "../common/components/ChatWidget/_Common/NullableImage";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const { getByTestId } = render(<Nullable />);
  const nullable = getByTestId("nullable-image");
  expect(nullable).toBeTruthy();
});

it("matches snapshot", () => {
  const tree = renderer.create(<Nullable />).toJSON();
  expect(tree).toMatchSnapshot();
});
