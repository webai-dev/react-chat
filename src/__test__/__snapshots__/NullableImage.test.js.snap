// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<img
  alt="nullable"
  className=""
  data-testid="nullable-image"
  onError={[Function]}
  src="Person.png"
/>
`;
